tests/unit/services/cookies-test.js

```diff
@@ -235,18 +237,27 @@ describeModule('service:cookies', 'CookiesService', {}, function() {
 
   describe('in the FastBoot server', function() {
     beforeEach(function() {
+
+      let request = EmberOject.extend({
+        init() {
+          this._super(...arguments);
+          this.cookies = {};
+          this.headers = {
+            append() {}
+          };
+        },
+        host: computed(function() {
+          return this._host;
+        })
```

Comment:
Unfortunately not. Using just host would work fine, the problem is that it works too much. :D 

When using just host as a plain property, the tests passes even without the fix of deconstructing.